.github/workflows/release-and-publish.yml

```diff
@@ -0,0 +1,55 @@
+name: Draft Release & Publish
+on: [workflow_dispatch]
+
+jobs:
+  Release-Artifacts:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+    # Checkcout code
+    - name: Checkout
+      uses: actions/checkout@v2
+    # Read cahngelog and read versions etc.
+    - name: Get current release version
+      id: package_version
+      uses: martinbeentjes/npm-get-version-action@v1.1.0
+    - name: Check version is mentioned in Changelog
+      id: changelog_reader
+      uses: mindsers/changelog-reader-action@v2
+      with:
+        validation_depth: 10
+        version: ${{ steps.package_version.outputs.current-version }}
+        path: 'CHANGELOG.md'
+    # Set the DRAFT_VERSION variable
+    - name: Set the draft version value
+      id: step_one
+      run: |
+        echo "DRAFT_VERSION=${{ steps.package_version.outputs.current-version }}" >> $GITHUB_ENV
+    - name: Check the value is set for draft version
+      id: step_two
+      run: |
+        echo "${{ env.DRAFT_VERSION }}" # This will output to the env var
+    # Create release tag out of Changelog entry
+    - name: Create a Release
```

Comment:
This could be moved to right under the other release step? 

If building fails we probably shouldn't create a tag

Reply:
💡 I think, Its part of one job so if something fails everything will fail. Kind of tested in other repos. We will test this again with fail tests.

Reply:
Typically if a step fails in a job then steps after it don't run. Steps run prior to the failing step aren't affected. There's ways around with the step if statement though

Reply:
👍

Reply:
👍